src/App/App.tsx

```diff
@@ -0,0 +1,48 @@
+import styles from './App.module.scss';
+import NavBar from 'App/components/NavBar';
+import { Outlet } from 'react-router-dom';
+import { useState, createContext, Dispatch, SetStateAction } from 'react';
+
+export interface Recipe {
+  documentId: number,
+  name: string,
+  calories: number,
+  summary: string,
+  images: Images[],
+  totalTime: number
+}
+
+export interface Images {
+  url: string
+}
+
+export interface ApiContextType {
+  recipes: Recipe[];
+  setRecipes: Dispatch<SetStateAction<Recipe[]>>;
+}
+
+export const ApiContext = createContext<ApiContextType>({
+  recipes: [],
+  setRecipes: () => { }
+});
+
+function App() {
```

Comment:
```suggestion
const App = ():React.FC => {
```

давай везде пропишем типы